upgrade-workspace.sh

```diff
@@ -185,6 +185,11 @@ create_secret() {
   fi
 }
 
+move_custom_service_folders() {
```

Comment:
Does this function need to be called in main()?

Reply:
Oops